### ISSUE TITLE
Doesn't work in an 'Amazon Household' that you don't own.

### ISSUE THREAD

**Issue by fergyfresh** [stevenleeg/geemusic#160]
There is something called Amazon Household that allows you to have multiple Amazon accounts have access to an Echo. When I disconnected my girlfriends phone and it was only my phone connected this whole setup worked great, was able to stream music to my device using the `ask gee music` prompts. Once my girlfriend reauthorized with the Amazon Household feature, it stopped working. It is almost like the Amazon Household doesn't share dev apps. This is troubling.

Only workaround I have come up with is to make me the leader of the Amazon Household, or to make the app instance on her account as well. Hmmm, this probably can't be fixed here can it?

**Comment by fergyfresh** [comment]
Added the owner of the household to the developer account for my Alexa Skill for Gee Music. Will post results tonight.

**Comment by fergyfresh** [comment]
Figured it out. If you are in an Amazon household with another Alexa user you must be the registered user for the device.

For example, if the two email addresses were `john.doe@example.com` and `jane.frank@example.com` and I were John Doe the Amazon Alexa skill would have to be created under my `john.doe@example.com` email address and the device when navigating to it on my Alexa app would have to say `device registered to: john.doe@example.com`.

Since my girlfriend plays Spotify from her phone I needed to add the skill under her developer account instead of mine.

**Comment by gabrielbergeron** [comment]
I had the same problem and I found too that adding the owner of the alexa device as a developer works. Even if I was in the household.

**Comment by fergyfresh** [comment]
Yeah. It ONLY works when the following statements are satisfied, the owner of the device is the owner of the amazon household and the owner of the household is the developer account that geemusic is setup on.
